src/main.ts

```diff
@@ -9,7 +9,7 @@ function throwError(error: Error): never {
 }
 
 async function bootstrap(): Promise<void> {
-  const app = await NestFactory.create(ApplicationModule, { cors: true });
+  const app = await NestFactory.create(ApplicationModule, { cors: true, logger: ['debug', 'verbose','warn','error','log'] });
```

Comment:
i think logger option is not needed in this app.
how about you?
and i want to your opinion about use all logger level.